doc/devdocs/types.rst

```diff
@@ -319,9 +319,11 @@ the type, which is an object of type :obj:`TypeName`:
      primary: Array{T,N} <: DenseArray{T,N}
      cache: SimpleVector
        ...
+
      linearcache: SimpleVector
        ...
-     uid: Int64 47
+
+     uid: Int64 -7900426068641098781
```

Comment:
Let me check out this branch on linux and see if it's consistent. I usually do most of the release preparation from Linux, with the exception of code signing the Windows binaries.